src/composer.ts

```diff
@@ -419,6 +420,42 @@ export class Composer<C extends Context> implements MiddlewareObj<C> {
         );
     }
 
+    /**
+     * Registers some middleware for certain chat types only. For example, you
+     * can use this method to only receive updates from private chats. The four
+     * chat types are `"channel"`, `"supergroup"`, `"group"`, and `"private"`.
+     *
+     * ```ts
+     * // Private chats only
+     * bot.chatType("private", ctx => { ... });
+     * // Channels only
+     * bot.chatType("channel", ctx => { ... });
+     * // Groups and supergroups only
+     * bot.chatType(["group", "supergroup"], ctx => { ... });
+     * ```
+     * Note that this combines very well with other filtering logic. This is how
+     * can you respond to `/start` commands from private chats:
+     * ```ts
+     * bot.chatType("private").command("start")
```

Comment:
I think there arent many usecases for using the middleware directly? Normally it will be a combination of filters like command, callbackQuery and so on. So the docs here should prefer using other commands rather than showing the usage with the middleware?

Reply:
I am not 100 per cent sure what you mean, are you suggesting to put this example higher up in the JSDoc comment, and skip the part about using `chatType` on its own?

Reply:
Most people will use it like this: `bot.chatType("private").command("start")`.

I don't see a real use case to use it like this: `bot.chatType("private", ctx => { ... })`. (Maybe except [stuff like this](https://github.com/HAWHHCalendarBot/TelegramBot/blob/3f9753c567694751006ef30ffc2101e3951d4388/source/parts/easter-eggs.ts#L15-L19))

I think the documentation should show prominently ("higher up") what most users will want to use.

Reply:
Good point, I'll fix it